plasma_framework/docs/design/payment-game-implementation-v1.md

```diff
@@ -105,14 +106,23 @@ For a Payment transaction to be exitable on standard exit, it only needs to chec
 1. Shows there exists valid spending tx of the input/output
 1. Removes the piggybacked flag of the challenged input or output
 
+### Delete non piggybacked In-flight Exit
+1. Checks that it is not in the first phase
+1. Checks that no piggyback on the in-flight exit
+1. Delete the in-flight exit and transfer IFE bond back to owner
+
 ### Process In-flight exit:
-1. Checks that none of the inputs is finalized. If any inputs is finalized, omit to process exit
-1. Checks the in-flight exit is canonical or not. Canonicity decides whether it should withdraw outputs or inputs
-1. To withdraw:
+1. Checks the in-flight exit is canonical or not. Canonicity decides whether it should withdraw outputs or inputs. An in-flight exit would be considered non-canonical when:
+   1. The canonicity games end up to be non-canonical
```

Comment:
`canonicity games` what are those. Can we specific which in-flight exit flows are related?